src/PKSim.Core/Snapshots/Mappers/OutputIntervalMapper.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Threading.Tasks;
+using SnapshotOutputInterval = PKSim.Core.Snapshots.OutputInterval;
+using ModelOutputInterval = OSPSuite.Core.Domain.OutputInterval;
+
+
+namespace PKSim.Core.Snapshots.Mappers
+{
+   public class OutputIntervalMapper : ParameterContainerSnapshotMapperBase<ModelOutputInterval, SnapshotOutputInterval>
```

Comment:
Special mapper for this one so that I can use the parameter mapping implementation from base. Nothing to do here